src/js/index.mjs

```diff
@@ -0,0 +1,54 @@
+export * as constants from './constants.mjs';
+export { default as labels } from './labels/labels.mjs';
+export * as timeUtils from './utils/time.mjs';
+import MediaAirplayButton from './media-airplay-button.mjs';
+import MediaCastButton from './media-cast-button.mjs';
+import MediaChromeButton from './media-chrome-button.mjs';
+import MediaGestureReceiver from './media-gesture-receiver.mjs';
+import MediaController from './media-controller.mjs';
+import MediaChromeRange from './media-chrome-range.mjs';
+import MediaControlBar from './media-control-bar.mjs';
+import MediaDurationDisplay from './media-duration-display.mjs';
+import MediaTimeDisplay from './media-time-display.mjs';
+import MediaCaptionsButton from './media-captions-button.mjs';
+import MediaSeekForwardButton from './media-seek-forward-button.mjs';
+import MediaFullscreenButton from './media-fullscreen-button.mjs';
+import MediaLiveButton from './media-live-button.mjs';
+import MediaMuteButton from './media-mute-button.mjs';
+import MediaPipButton from './media-pip-button.mjs';
+import MediaPlayButton from './media-play-button.mjs';
+import MediaPlaybackRateButton from './media-playback-rate-button.mjs';
+import MediaPosterImage from './media-poster-image.mjs';
+import MediaSeekBackwardButton from './media-seek-backward-button.mjs';
+import MediaPreviewTimeDisplay from './media-preview-time-display.mjs';
+import MediaPreviewThumbnail from './media-preview-thumbnail.mjs';
+import MediaTimeRange from './media-time-range.mjs';
+import MediaLoadingIndicator from './media-loading-indicator.mjs';
+import MediaVolumeRange from './media-volume-range.mjs';
```

Comment:
the source files should really keep the `.js` extension. 
we can compile to anything but this is unneeded IMO.